mucommander-core/src/main/java/com/mucommander/ui/main/table/FileTable.java

```diff
@@ -858,6 +858,10 @@ public void sortBy(Column criterion) {
         if (criterion==sortInfo.getCriterion()) {
             reverseSortOrder();
             return;
+        } else {
+            // The selected column is different from the previously selected one.
+            // In this case, default to ascending sort order
+            sortInfo.setAscendingOrder(true);
```

Comment:
how about calling `sortBy` here with ascending==true instead? (and remove the `return` statement above which becomes redundant)

Reply:
In that case I will have to duplicate the next line (call to `sortBy` at line 867) - was that your intention?

Reply:
not to duplicate but to replace the call to sortBy at line 867

Reply:
ohhh right - got you. let me make the change

Reply:
I made the change and updated the readme file as well. Thanks for your feedback!